screens/rating.js

```diff
@@ -0,0 +1,160 @@
+import React from 'react';
+import { StyleSheet, Text, View, Dimensions, Image} from 'react-native';
+import { Font } from 'expo';
+
+const window = Dimensions.get("window")
+
+export default class Login extends React.Component {
+  state = {
+    fontLoaded: false,
+  };
+  async componentDidMount() {
+    await Font.loadAsync({
+      'Montserrat-Regular': require('/Users/nona/shopu/assets/fonts/Montserrat-Regular.otf'),
```

Comment:
Again, change the paths of your fonts

Reply:
When I change the paths and reload the app, it says it's unable to find the fonts. Do I have to put in exactly `./../assets/fonts/Montserrat-Regular.otf`?

Reply:
No, that was just an example

When I was walking through bash commands, I mentioned that `.` is the current directory and `..` is the parent directory.

Reply:
I got it to work! Thanks!